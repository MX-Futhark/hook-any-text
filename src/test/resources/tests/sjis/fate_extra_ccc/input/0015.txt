E9 78 82 B3 82 EA 82 C8 82 A2 82 C5 82 B2 8E E5 90 6C 97 6C 81 42 0A 82 A0 82 CC 8F CE 8A E7 81 41 88 AB 8D 49 82 DD 82 AA 82 A4 82 DC 82 AD 82 A2 82 C1 82 BD 8E 9E 82 CC 0A 8F 97 82 CC 8F CE 8A E7 82 C5 82 B7 82 A9 82 E7 81 42 00 FF FF FF 00 00 00 00 00 01 00 00 00 02 00 00 00 03 00 00 00 04 00 00 00 05 00 00 00 06 00 00 00 07 00 00 00 08 00 00 00 09 00 00 00 0A 00 00 00 0B 00 00 00 0C 00 00 00 0D 00 00 00 0E 00 00 00 0F 00 00 00 10 00 00 00 11 00 00 00 12 00 00 00 13 00 00 00 14 00 00 00 15 00 00 00 16 00 00 00 17 00 00 00 18 00 00 00 19 00 00 00 1A 00 00 00 1B 00 00 00 1C 00 00 00 1D 00 00 00 1E 00 00 00 1F 00 00 00 20 00 00 00 21 00 00 00 22 00 00 00 23 00 00 00 24 00 00 0B 00 00 00 BC 14 C9 08 00 00 00 00 00 00 00 00 00 00 00 00 00 00 5C 00 00 00 00 00 00 00 00 00 FF FF FF DC FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 00 00 A8 42 FF FF FF DC FF FF FF FF 00 00 50 43 00 00 00 00 00 00 F0 43 00 00 A8 42 FF FF FF DC FF FF FF FF 00 00 00 00 00 00 88 43 00 00 00 00 0B 00 00 00 00 00 00 00 FF FF FF FF 00 00 50 43 00 00 88 43 03 00 10 00 00 00 00 00 0B 00 00 00 3C 15 C9 08 00 00 00 00 00 00 00 00 00 00 00 00 02 00 5C 00 00 00 00 00 00 00 00 00 FF FF FF DC FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 00 00 18 43 FF FF FF DC FF FF FF FF 00 00 50 43 00 00 00 00 00 00 18 43 00 00 18 43 FF FF FF DC FF FF FF FF 00 00 00 00 00 00 88 43 00 00 00 00 0B 00 00 00 A0 15 C9 08 FF FF FF FF 00 00 50 43 00 00 88 43 04 00 10 00 00 01 00 00 02 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 02 00 C2 01 00 00 00 00 78 F9 A2 48 0B 00 00 00 5C 17 C9 08 00 00 00 00 00 00 00 00 00 00 00 00 02 00 10 00 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF DC 00 00 00 00 00 00 3C 43 00 00 00 00 00 00 F0 43 00 00 00 00 FF FF FF DC 00 00 F0 43 00 00 3C 43 00 00 00 00 00 00 00 00 00 00 A8 42 FF FF FF DC 00 00 00 00 00 00 88 43 00 00 00 00 00 00 F0 43 00 00 A8 42 FF FF FF DC 00 00 F0 43 00 00 88 43 00 00 00 00 00 00 00 00 0B 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 03 00 10 00 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF DC 00 00 60 C1 00 00 1F 43 00 00 00 00 00 00 18 43 00 00 00 00 FF FF FF DC 00 00 0A 43 00 00 1F 43 00 00 00 00 00 00 00 00 00 00 18 43 FF FF FF DC 00 00 60 C1