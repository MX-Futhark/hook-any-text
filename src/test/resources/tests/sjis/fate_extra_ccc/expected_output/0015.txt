["騙されないでご主人様。
あの笑顔、悪巧みがうまくいった時の
女の笑顔ですから。"]